libraries/c2.js.yaml

```diff
@@ -0,0 +1,14 @@
+name: c2.js
+category: math, physics, algorithms
```

Comment:
Thanks for your submission! Right now we are placing libraries in a single category, and that will be the heading it will be under on the libraries page. Is there a single one of these that you feel would fit best?

Reply:
algorithms